setup/install.py

```diff
@@ -234,6 +235,22 @@ def systemd(self, width=80):
             '--verbose',
             action='store_true',
             help='Enable verbose mode.')
+    
+    def docker(self, width=80):
```

Comment:
Please update the documentation regarding the new commands to install using docker.